src/thirdparty/query-string/index.js

```diff
@@ -0,0 +1,221 @@
+const strictUriEncode = require('../strict-uri-encode');
```

Comment:
I had to do some weird stuff to get your build fixed. Will explain why tomorrow.